components/kubeflow/deployer/src/deploy.sh

```diff
@@ -95,6 +100,13 @@ echo "Generating the TF Serving config..."
 ks generate tf-serving server --name="${SERVER_NAME}"
 ks param set server modelPath "${MODEL_PATH}/export/export"
 
+# support local storage to deploy tf-serving.
+# TODO: Remove modelStorageType setting after enhanced hardcode in kubeflow/tf-serving/tf-serving.libsonnet. 
```

Comment:
I'd like to see the detailed code link in the comments, like you can use the link as this: https://github.com/kubeflow/kubeflow/blob/v0.4-branch/kubeflow/tf-serving/tf-serving.libsonnet#L148-L151

Reply:
@gyliu513 Done. Thanks a lot!